
FAIL navigate event destination.index should be dynamic promise_test: Unhandled rejection with value: object "TypeError: undefined is not an object (evaluating 'back_destination.index')"

